### PR TITLE
style: format files

### PR DESCRIPTION
**Pull request by marfavi**
run `dotnet format` in the `coffeecard` dictionary

### CODE REVIEW

File: coffeecard/CoffeeCard.WebApi/appsettings copy.bicep [hunk 1]
```diff
@@ -0,0 +1,177 @@
+// {
```
Comment (TTA777) [inline coffeecard/CoffeeCard.WebApi/appsettings copy.bicep:1]: Where did this file come from?

**Review by TTA777 — CHANGES_REQUESTED** [review]
All good, except one mystery file

**Comment by jonasanker** [comment]
Great initiative! Maybe it could be useful to add a pipeline CI step where we verify that all files are formatted. Similar to what we are doing in the coffeecard_app repository.

The command
```
dotnet format --verify-no-change
```

https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format#options

File: coffeecard/CoffeeCard.WebApi/appsettings copy.bicep (hunk 1 above)
Comment (marfavi) [reply coffeecard/CoffeeCard.WebApi/appsettings copy.bicep:1]: probably leftover file from the last time i was in this repo. removing

**Comment by marfavi** [comment]
> Great initiative! Maybe it could be useful to add a pipeline CI step where we verify that all files are formatted. Similar to what we are doing in the coffeecard_app repository.
> 
> The command
> 
> ```
> dotnet format --verify-no-change
> ```
> 
> https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format#options

#180

**Pull request merged**